dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -0,0 +1,147 @@
+[[getting-traffic-into-cluster]]
+= Getting Traffic Into The Cluster
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+== Overview
+There are many ways to access the cluster. This section describes some
+commonly used approaches.
+
+The recommendation is:
+
+- If you have HTTP/HTTPS -- Use the xref:../dev_guide/getting_traffic_into_cluster.adoc#using-a-router[Router],
+- If you have Encrypted TCP (i.e., TLS with the SNI header) -- Use the xref:../dev_guide/getting_traffic_into_cluster.adoc#using-a-router[Router],
+- If you have TCP or UDP -- Use xref:../dev_guide/getting_traffic_into_cluster.adoc#using-the-loadbalancer[LoadBalancer],
```

Comment:
Can we add some of the text from my comment explaining why you might want one of the three over the others?